index.html

```diff
@@ -0,0 +1,143 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title></title>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="assets/styles/css/styles.css" />
+  </head>
+
+  <body>
+    <div class="wrapper">
+      <div class="nav">
```

Comment:
Better to use semantic tags. In this case, the Header tag is more appropriate. And next in header to use nav tag then ul -li